Sources/JWT/JWTError.swift

```diff
@@ -4,6 +4,7 @@ public enum JWTError: Error {
     case decoding
     case encoding
     case incorrectNumberOfSegments
+    case incorrectNodeType
```

Comment:
This is technically a breaking change. :( There should be an `undefined` or `unknown` type on this enum that we can use. If not, we need to use one of the existing errors.